src/Identity/v3/Api.php

```diff
@@ -709,6 +710,15 @@ public function getCredentials(): array
         ];
     }
 
+    public function getUserCredentials(): array
```

Comment:
There is not need to create a new definition there. We already have `getCredentials()` and can just reuse it by adding the `userId` parameter.